components/create_team/create_team.jsx

```diff
@@ -6,15 +6,30 @@ import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import {Route, Switch, Redirect} from 'react-router-dom';
 
-import ChannelStore from 'stores/channel_store.jsx';
-import TeamStore from 'stores/team_store.jsx';
 import AnnouncementBar from 'components/announcement_bar';
 import BackButton from 'components/common/back_button.jsx';
 
-import TeamUrl from '../team_url';
-import DisplayName from '../display_name';
+import TeamUrl from 'components/create_team/components/team_url';
+import DisplayName from 'components/create_team/components/display_name';
+
+export default class CreateTeam extends React.PureComponent {
+    static propTypes = {
+
+        currentTeam: PropTypes.object,
+
+        currentChannel: PropTypes.object,
+
+        isLicensed: PropTypes.bool.isRequired,
+
+        customBrand: PropTypes.bool.isRequired,
+
+        enableCustomBrand: PropTypes.bool.isRequired,
+
+        customDescriptionText: PropTypes.string,
+
+        siteName: PropTypes.string
+    }
```

Comment:
Could you please add comments to prop types?

Reply:
Definitely can do, but I can't say I fully understand the difference between `customBrand` and `enabledCustomBrand`. With both being boolean types I can't help but feel like they're redundant (I searched for comments explaining them all over the project but couldn't find any).

My interpretation is that `customBrand` represents whether the user is connected to a server that has an enterprise version that supports custom branding, and `enabledCustomBrand` is about whether the feature has been actually enabled in the System Console. Is that right or am I missing something?

Reply:
Yes, you're correct.